terraform/modules/apps/publishing-api/variables.tf

```diff
@@ -65,3 +76,22 @@ variable "govuk_publishing_platform_namespace_id" {
 variable "govuk_publishing_platform_namespace_name" {
   type = string
 }
+
+variable "govuk_website_root" {
+  type = string
+}
+
+variable "redis_host" {
+  description = "FQDN of the redis cluster for Publishing API"
+  type        = string
+}
+
+variable "redis_port" {
+  description = "Port of the Redis Cluster for Publishing API"
+  type        = number
+  default     = 6379
```

Comment:
Duplicate definition of the default in govuk/variables.tf - best delete this one.
```suggestion
```